src/component-library/utils/decives.ts

```diff
@@ -0,0 +1,15 @@
+const sizes = {
+  mobile: '320px',
```

Comment:
I'm not keen on breakpoints named after devices - they tend to imply a relationship between a screen size and a device that isn't always there. I think there are probably too many breakpoints as well - based on our designs, I would expect there to be 3 major breakpoints, with additional specific breakpoints brought in at the component level if and when needed. Let's have a chat about how we handle this.